coreMacros/src/main/scala/Chisel/internal/sourceinfo/SourceInfoTransform.scala

```diff
@@ -6,7 +6,7 @@
 package Chisel.internal.sourceinfo
 
 import scala.language.experimental.macros
-import scala.reflect.macros.blackbox.Context
+import scala.reflect.macros.whitebox.Context
```

Comment:
Give that whitebox macros are outside the norms (and trickier), it would be best to make it more explicit which macros are using them.

Instead import scala.reflect.macros.whitebox
Then, declare as whitebox.Context instead of just Context.